intuita-webview/src/codemodList/CodemodNodeRenderer/style.module.css

```diff
@@ -44,6 +44,10 @@
 	align-items: center;
 }
 
+.labelContainer.focused {
```

Comment:
can we use the https://developer.mozilla.org/en-US/docs/Web/CSS/:focus pseudo-class here?

Reply:
Nope. We are not handling pseudo classes well in the app. Good point because this is something we use improve for better a11y